### PR TITLE
Synchronize with LDM-230 and LDM-129. 

### PR DESCRIPTION
**Pull request by ktlim**
Product naming should now be consistent with `productlist.csv` in LDM-294.

### CODE REVIEW

File: LDM-148.tex [hunk 1]
```diff
@@ -171,23 +171,22 @@ \section{Component Overview}\label{component-overview}
 components. The top tier is the LSST Science Platform, which is deployed
 in the DACs and other computational environments to provide a user
 interface and analysis environment for science users and LSST staff. The
-detailed design of this tier is given in ``LSST Science Platform,'' \citeds{LDM-542}. The next
+detailed design of this tier is given in ``LSST Science Platform'' (\citeds{LDM-542}). The next
```
Comment (timj) [inline LDM-148.tex]: The other option here, and in other places in the document, is to use `\citedsp{}`

File: LDM-148.tex [hunk 2]
```diff
@@ -414,50 +405,73 @@ \subsubsection{Image and EFD Archiving}\label{image-and-efd-archiving}
...
+Archiver and Catch-Up Archiver to Observatory Operations Data Service:
+image files with associated metadata are written to OODS storage.
+
+Observatory Operations Data Service to Data Backbone: files are copied to
 Data Backbone storage via a file transfer mechanism, and their
 information and metadata are registered with Data Backbone management
 dataabases. The Data Butler is not used for this low-level,
 non-science-payload interface.
 
+Observatory Operations Data Service to Commissioning Cluster LSP and other users: the OODS provides a mountable POSIX filesystem interface. A web interface (e.g. WebDAV) may also be provided, but this will be as simple as possible to enable maintaining a very high level of service availability and reliability.
```
Comment (timj) [inline LDM-148.tex]: Line is longer than one sentence.

File: LDM-148.tex [hunk 3]
```diff
@@ -414,50 +405,73 @@ \subsubsection{Image and EFD Archiving}\label{image-and-efd-archiving}
...
 environments.
 
 \subsection{Interfaces}\label{base-interfaces}
 
-OCS to all Base Enclave services: these interface through the SAL
+OCS to all Base L1 Enclave services: these interface through the SAL
 library provided by the OCS subsystem.
 
-Archiver and Catch-Up Archiver to Data Backbone: files are copied to
+Archiver and Catch-Up Archiver to Observatory Operations Data Service:
```
Comment (timj) [inline LDM-148.tex]: Add `(OODS)` here to indicate that that is what later usage of OODS means.

Comment (ktlim) [reply LDM-148.tex]: The abbreviation is defined above in line 395 and has already been used twice by this point.

**Comment by ktlim** [comment]
The use of hard-broken lines was to minimize the diffs with the previous version.  I have now sentence-broken areas that were moved or otherwise rewritten significantly, so they showed up as large diff blocks anyway.

**Pull request merged**